config/routes.rb

```diff
@@ -113,7 +113,7 @@
     resource(:miscellaneous_np_info, {only: [:show, :update]})
 
 		namespace(:button) do
-			root({to: :advanced})
+			root({action: :advanced})
```

Comment:
I did this tiny change because this deprecation warning was very annoying and it was easy to resolve, but I can do it on other PR if you prefer :smile_cat:

Reply:
Another PR would be ideal.